Databricks/Dockerfile

```diff
@@ -109,17 +109,27 @@ RUN chmod 755 /opt/spark-rapids/init.sh
 
 WORKDIR /databricks
 
+FROM ubuntu:18.04 as download-modpython
```

Comment:
weird to pull from 18.04 when base is 20.04.. add comment

Reply:
Added the comment.